src/Services/Microsoft.AspNetCore.SystemWebAdapters.Services.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nit-pick - maybe change the folder name to match the project name in order to match the pattern of other projects in this solution.